src/components/AvatarWithIndicator.js

```diff
@@ -46,8 +44,8 @@ const propTypes = {
 const defaultProps = {
     size: 'default',
     tooltipText: '',
-    policiesMemberList: {},
     policies: {},
+    policiesMemberList: {},
```

Comment:
Sorry to move this around, it's the result of fixing merge conflicts.

Reply:
FWIW you could also have put it back where it was 😄 (NAB of course)